### PR TITLE
Update notebook and upload fonts

### PR DESCRIPTION
**Pull request by EEdwardsA**
Notebook updated with markdown and commenting for benefit of next team
Fonts uploaded

# Description

Replace with description of PR

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

**Review by jproctor-rebecca — APPROVED** [review]
Thank you for adding/updating all of the font licensing information. I know this is tedious.  However, it is far more time-consuming for someone to come behind us to find this information.  Excellent practice!

File: notebooks/generate_synthetic_data.ipynb [hunk 1]
```diff
@@ -33,165 +32,11 @@
...
-   ],
-   "source": [
-    "patoolib.extract_archive('./fonts/acki-preschool.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/bpchildlefty.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/schoolbell.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/tommys-first-alphabet.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/acki-preschool.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/two-turtle-doves.zip',outdir='./fonts/')"
+    "You will probably want to play around with this sample to better understand a sample image's parameters before you later go on to tweak the sample generating function (which randomizes across a large space of possible parameters)."
    ]
```
Comment (jproctor-rebecca) [inline notebooks/generate_synthetic_data.ipynb:40]: Adding the section above explains thy why behind the code and clarifies your processes.  Thank you for including.

File: notebooks/generate_synthetic_data.ipynb [hunk 2]
```diff
@@ -210,23 +54,24 @@
     "id": "5PlLJJuOI-0A"
    },
    "source": [
-    "I googled for tutorials and it seems like there's a Python library called Pillow for working with images, including adding text to images with fonts.\n",
-    "\n",
-    "Based on some tutorials, here's a working example using the font we downloaded."
```
Comment (jproctor-rebecca) [inline notebooks/generate_synthetic_data.ipynb:215]: Do you have the links to these tutorials so I can include them in the resources list I am compiling?

File: notebooks/generate_synthetic_data.ipynb [hunk 3]
```diff
@@ -346,41 +193,158 @@
...
+    "- Restricting stories to \"General Audiences\" maturity level only\n",
+    "- Restricting stories to \"No Archive Warnings Apply\"\n",
+    "- Restricting stories to \"Gen\" i.e. no content focused on romantic relationships\n",
+    "\n",
+    "There are large selection effects for which fanfiction archive a writer publishes their stories in, several of which correlate with age. Archive of Our Own, the current source of scraped data, probably has the overall oldest/most mature userbase. Alternate archives you could scrape, which have younger and more amateur user bases, are fanfiction.net and Wattpad.\n",
+    "\n",
+    "Here is a web scraper script for fanfiction.net that you can modify: https://github.com/smilli/fanfiction\n",
+    "\n",
+    "I have not found a web scraper for Wattpad."
    ]
```
Comment (jproctor-rebecca) [inline notebooks/generate_synthetic_data.ipynb:343]: Thorough description of logic here.  Thank you.

File: notebooks/generate_synthetic_data.ipynb [hunk 4]
```diff
@@ -429,20 +390,101 @@
...
+    "        \n",
+    "        flip = random.randint(0,1)\n",
+    "        if flip:\n",
+    "            pass\n",
+    "#         draw.line((0,50+font_size+offset, width,50+font_size+offset), fill='black')\n",
+    "        for j in range(50+font_size+offset, height, int(height/(font_size+offset))):\n",
+    "            draw.line((0, j, width, j), fill='black')\n",
+    "        \n",
+    "        img.save(f'./synth_images/img{i}','jpeg')\n"
    ]
```
Comment (jproctor-rebecca) [inline notebooks/generate_synthetic_data.ipynb:488]: naming these recommendations and clarifying the why behind them are important transfers of knowledge to the incoming team; excellent thinking!

**Review by jproctor-rebecca — APPROVED** [review]

**Review by James-Hagerman — APPROVED** [review]

**Review by trevorjamesmartin — APPROVED** [review]

**Pull request merged**